docs/README_UA.md

```diff
@@ -0,0 +1,541 @@
+Монгуастичний
+=============
+
+
+Mongoosastic - це плагін [mongoose] (http://mongoosejs.com/), який може автоматично індексувати ваші моделі в [elasesearch] (https://www.elastic.co/).
+
+- [Установка] (# установка)
+- [Налаштування] (# налаштування)
+- [Індексація] (# індексація)
+  - [Збереження документа] (# збереження документа)
+  - [Видалення документа] (# видалення-документа)
+  - [Індексація вкладених моделей] (# індексація-вкладених моделей)
+  - [Індексація посилань на мангустів] (# індексація-мангуста-посилання)
+  - [Індексація існуючої колекції] (# індексація-наявна колекція)
+  - [Масова індексація] (# об'ємна індексація)
+  - [Фільтрована індексація] (# відфільтрована-індексація)
+  - [Індексація попиту] (# індексація за запитом)
+  - [Скасування вилучення за запитом] (# видалення за запитом)
+  - [Обрізання індексу] (# truncating-an-index)
+  - [Обмеження] (# обмеження)
+    - [Автоіндексація] (# автоматична індексація)
+    - [Пошук відразу після es-індексованої події] (# search-відразу-після-es-indexed-event)
+- [Картографування] (# відображення)
+  - [Географічне відображення] (# географічне відображення)
+    - [Індексація геоточки] (# індексація-гео-точка)
+    - [Індексація геофігури] (# індексація-гео-форма)
+  - [Створення відображень за запитом] (# створення-відображення на вимогу)
+- [Запити] (# запитів)
+  - [Гідратація] (# гідратація)
+
+## Встановлення
+
+Остання версія цього пакета буде максимально наближена до найновіших пакетів `elasticsearch` та` mongoose`.
+
+`` `баш
+npm встановити -S монгоосастичний
+`` `
+
+## Налаштування
+
+### Model.plugin (монгоосастика, параметри)
```

Comment:
This a header, but also a function signature, please leave it as is, and maybe add the translation on the section body.